package.json

```
@@ -33,8 +33,8 @@
     "lint": "NODE_ENV=production eslint --quiet --ext .tsx,.ts,.js .",
     "lint:fix": "yarn lint --fix",
     "prettier:check": "prettier --check \"**/*.{json,html,css,scss}\"",
-    "playground": "cd playground && webpack-dev-server",
-    "playground:ie": "cd playground && webpack-dev-server --host=0.0.0.0 --disable-host-check --useLocalIp",
+    "playground": "cd playground && webpack serve --port=9888",
```

ugh sorry forgot I added the port

```suggestion
    "playground": "cd playground && webpack serve",
```